website/pages/en/sunrise.mdx

```diff
@@ -46,7 +46,7 @@ The Graph community is here to support developers as they move to The Graph Netw
 
 ### How can I ensure high quality of service and redundancy for subgraphs on The Graph Network?
 
-All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
+All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph and GRT](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```

Comment:
"adding signal to your subgraph and GRT"? I don't get the phrasing. 🤔

Reply:
Existed before where? I'm confused. Looks like you added "and GRT".

Reply:
Hi! Sorry, I confused it with another docs page phrasing.

Reply:
No worries :) I see you removed it.